cinema/serializers.py

```diff
@@ -0,0 +1,19 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.Serializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(required=True, max_length=100)
+    description = serializers.CharField(required=False, max_length=100)
```

Comment:
The `description` field's `max_length` is set to 100, but in the `Movie` model, it is 255. Consider updating the `max_length` to 255 to match the model definition.